pkg/bitwarden/cli.go

```diff
@@ -93,12 +95,29 @@ func (c *cliClient) GetFieldOnItem(itemName, fieldName string) ([]byte, error) {
 	return nil, fmt.Errorf("failed to find field %s in item %s", fieldName, itemName)
 }
 
-func (c *cliClient) GetAttachmentOnItem(itemName, attachmentName string) ([]byte, error) {
+func (c *cliClient) GetAttachmentOnItem(itemName, attachmentName string) (bytes []byte, retErr error) {
+	file, err := ioutil.TempFile("", "attachmentName")
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		if err := os.Remove(file.Name()); err != nil {
+			retErr = err
```

Comment:
defer will run after the function returns. `retErr` will never get that value.

Reply:
It will work (return `retErr = err`) as we discussed in slack.